apple-pie/styles/style.css

```diff
@@ -0,0 +1,123 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+hr{
+    width: 65%;
+}
+
+ul{
+    list-style: none;
+    padding: 0;
+}
+
+ol {
+    list-style: none;
+  }
+  
+  ol > li {
+    counter-increment: listcounter;
+  }
+  
+  ol > li::before {
+    content: counter(listcounter) ")";
+  }
+
+  p{
+    text-align: center;
+    font-size: 25px;
+    font-style: normal;
+    color:rgb(37, 37, 37)
+  }
+
+  header {
+    background-image: url(/apple-pie/images/apple-pie.jpg);
+    position: relative;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size:100%;
+    margin: 0px, 0px;
+    padding: 0px, 0px;
+    margin-top: 0px;
+    display: block;
+    
+    }
+    header h1{
+        color: white;
+        text-align: center;
+        padding-top: 450px;
```

Comment:
If you added the previous line of code for your background image, now your title is not centered with the image, so you can lower a little bit your `padding` change it from 450px to 250px:
`padding-top: 250px;`